Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM node:8-slim
-ENV NPM_CONFIG_LOGLEVEL error
+#
+# --- Base Node Image ---
+FROM node:8-alpine AS base
+
 WORKDIR /src
-ADD . /src
-RUN cd /src \
- && npm install \
- && npm run build \
- && npm cache clear --force \
- && rm -rf ~/.npm \
- && rm -rf /var/lib/apt/lists/*
-ENTRYPOINT ["npm", "run", "dashboard"]
+
+# Copy package.json first to benefit from layer caching
+COPY package*.json ./
+RUN npm install --only=production
+# Copy production node_modules aside for later
+RUN cp -R node_modules prod_node_modules
+# Install remaining dev dependencies
+RUN npm install
+
+COPY . /src
+
+# Run all webpack build steps
+RUN npm run prepublish && npm run build
+
+
+#
+# --- Production Image ---
+FROM node:8-alpine AS release
+WORKDIR /src
+
+# Copy production node_modules
+COPY --from=base /src/prod_node_modules /src/node_modules
+COPY --from=base /src/package*.json /src/
+
+# Copy compiled src dirs
+COPY --from=base /src/Parse-Dashboard/ /src/Parse-Dashboard/
+
+ENTRYPOINT ["npm", "run", "start"]
```

Comment:
npm start is enough, no need to use `run`.

Also, I believe it is better to use node directly than using npm to start the server so we should use:

`node ./Parse-Dashboard/index.js` for the ENTRYPOINT

Reply:
I've updated the entrypoint and everything seems to still be working nicely :)